src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreTargetGraph.cs

```diff
@@ -85,16 +85,31 @@ public class RestoreTargetGraph : IRestoreTargetGraph
             ResolvedDependencies = resolvedDependencies;
         }
 
-        public static RestoreTargetGraph Create(IEnumerable<GraphNode<RemoteResolveResult>> graphs, RemoteWalkContext context, ILogger logger, NuGetFramework framework)
+        /// <summary>
```

Comment:
This looks to be unused in product code? Can we 🔪 it?

Reply:
It has 5 references. Do you mean to dig deeper an remove those references as well?

Reply:
I think I meant to say it's not used in product code. 

I think things that aren't used in product code would be great candidates to make internal at this poitn. Especially things are just for utility, rather than necessity.